testing/fuzzing/fuzz_support.cc

```diff
@@ -22,6 +22,8 @@
 
 const bool DEBUG = false;
 
+static constexpr tox_mono_time_cb *get_self_clock = ![](Fuzz_System *self) { return self->clock; };
```

Comment:
why `!` instead of `+` ?

(i feel like i have asked this before)

Reply:
No reason. I chose `!` when I implemented it. PR to change it is welcome if you feel it should be.